teamcity: failed tests on master: test/TestTrace, test/TestTrace/ShowTraceForSplitBatch, test/TestTrace/ShowTraceForSplitBatch/TracingOff, test/TestTrace/ShowTraceForSplitBatch/TracingOff/node-1

The following tests appear to have failed:

[#314172](https://teamcity.cockroachdb.com/viewLog.html?buildId=314172):

```
--- FAIL: test/TestTrace (40.210s)
test_log_scope.go:78: test logs captured to: /go/src/github.com/cockroachdb/cockroach/artifacts/logTestTrace698671528
test_log_scope.go:61: use -show-logs to present logs inline
--- FAIL: test/TestTrace/ShowTraceForSplitBatch (9.610s)
null
--- FAIL: test/TestTrace/ShowTraceForSplitBatch/TracingOff (5.530s)

------- Stdout: -------
E170803 16:09:22.107836 253993 storage/queue.go:656  [replicate,n1,s1,r1/1:/{Min-System/}] range requires a replication change, but lacks a quorum of live replicas (0/1)
E170803 16:09:22.112589 254012 storage/replica_proposal.go:538  [n1,s1,r2/1:/{System/-Max}] could not load SystemConfig span: must retry later due to intent on SystemConfigSpan
E170803 16:09:22.117747 254018 storage/replica_proposal.go:538  [n1,s1,r3/1:/{System/tsd-Max}] could not load SystemConfig span: must retry later due to intent on SystemConfigSpan
--- FAIL: test/TestTrace/ShowTraceForSplitBatch/TracingOff/node-1 (0.050s)
trace_test.go:246: expected span: "kv.DistSender: sending partial batch", got: "/cockroach.roachpb.Internal/Batch"
------- Stdout: -------
E170803 16:09:27.557415 253992 storage/queue.go:656  [split,n1,s1,r15/1:/Table/5{0-2/1/5}] unable to split [n1,s1,r15/1:/Table/5{0-2/1/5}] at key "/Table/51": unexpected value: raw_bytes:"~-\340\222\003\010\017\022\001\272\032\003\274\211\215\"\006\010\001\020\001\030\001\"\006\010\002\020\002\030\002\"\006\010\003\020\003\030\003(\004" timestamp:<wall_time:1501776566909752837 logical:1 > 

```
Please assign, take a look and update the issue accordingly.

--- a/pkg/sql/trace_test.go
+++ b/pkg/sql/trace_test.go
@@ -145,13 +145,12 @@ func TestTrace(t *testing.T) {
 				// and will split the underlying BatchRequest/BatchResponse. Tracing
 				// in the presence of multi-part batches is what we want to test here.
 				return sqlDB.Query(
-					"SELECT operation op FROM [SHOW TRACE FOR DELETE FROM test.bar] " +
+					"SELECT DISTINCT(operation) op FROM [SHOW TRACE FOR DELETE FROM test.bar] " +
 						"WHERE message LIKE '%1 DelRng%' ORDER BY op")
 			},
 			expSpans: []string{
 				"kv.DistSender: sending partial batch",
 				"starting plan",
-				"/cockroach.roachpb.Internal/Batch",
 				"/cockroach.roachpb.Internal/Batch",
 			},
 		},

